.github/workflows/unit-tests-check.yml

```diff
@@ -0,0 +1,39 @@
+name: Ensure Unit Tests Running
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  run_tests:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Required Packages
+      run: |
+        sudo apt-get update && sudo apt-get install libxmlsec1-dev
+        sudo systemctl start mongod
+
+    - name: Install pip
+      run: pip install -r requirements/edx/pip-tools.txt
```

Comment:
Shouldn't this be `requirements/pip.txt` instead?

Reply:
This wasn't needed. Removed this.